decidim/decidim
# Fix initiatives display when not initialized

#### :tophat: What? Why?
This PR removes the initiatives button in the menu when there is no initiative types or scope created.
It also redirects any request to initiatives to the homepage with a flash message

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #8935

#### Testing
On a clean installation
- see there is no button initiatives in the menu
- try access an initiative page
- see the redirection and the flash message
- 
:hearts: Thank you!
